File: ans/2/python/7.py
def my_series(num):
	first = 1
	second = 3

	num -= 2

	while(num > 0):
		first, second = second, first + second
		print("second = {0}".format(second))
		num -= 1

	return second

res = my_series(23)
print("res = {0}".format(res))

File: ans/2/c/7.c
#include <stdio.h>

int first = 1;
int second = 3;

int calc_custom_series(int no)
{
	int i, tmp;

	for(i = 0; i < no - 2; i++)
	{
		tmp = first;
		first = second;
		second = tmp + second;
		printf("res = %d\n", second);
	}

	return second;
}

int main(void)
{
	int res = calc_custom_series(23);
	printf("res = %d\n", res);

	return 0;
}
